### PR TITLE
Send Emails & Security

### PR DESCRIPTION
**Pull request by CapCap**
- Send templatized emails 
- Change email token on every send
- Send AIT1 registration finished email
- correct `sentry_scope.set_context` because their docs are exceptionally misleading

### CODE REVIEW

File: ecosystem/platform/server/app/jobs/kyc_complete_job.rb [hunk 1]
```diff
@@ -3,16 +3,15 @@
...
+class KYCCompleteJobError < StandardError; end
 
 class KYCCompleteJob < ApplicationJob
   # Ex args: { user_id: 32, inquiry_id=inq_syMMVRdEz7fswAa2hi }
   def perform(args)
     user = User.find(args[:user_id])
     sentry_scope.set_user(id: user.id)
+
     inquiry_id = args[:inquiry_id]
-    sentry_scope.set_context(:inquiry_id, inquiry_id)
```
Comment (zacharydenton) [inline ecosystem/platform/server/app/jobs/kyc_complete_job.rb:15]: No longer needed?

File: ecosystem/platform/server/app/controllers/it1_profiles_controller.rb [hunk 2]
```diff
@@ -43,6 +43,7 @@ def create
       log @it1_profile, 'created'
       validate_node(v, do_location: true)
       if @it1_profile.validator_verified?
+        current_user.maybe_send_ait1_registration_complete_email
```
Comment (zacharydenton) [inline ecosystem/platform/server/app/controllers/it1_profiles_controller.rb:46]: I know we say "Step 1" and "Step 2" but I guess they can be completed in any order?

**Review by zacharydenton — APPROVED** [review]
LGTM

File: ecosystem/platform/server/app/jobs/kyc_complete_job.rb (hunk 1 above)
Comment (CapCap) [reply ecosystem/platform/server/app/jobs/kyc_complete_job.rb:15]: ApplicationJob now automatically add context for all job args :-)

File: ecosystem/platform/server/app/controllers/it1_profiles_controller.rb (hunk 2 above)
Comment (CapCap) [reply ecosystem/platform/server/app/controllers/it1_profiles_controller.rb:46]: if someone is KYC exempt- they only need to do step 1. If someone edits their node registration, they'll have to re-verify it and they'll end up getting an email again

**Pull request closed without merging**